sdk/keyvault/keyvault-admin/.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "plugins": ["@azure/azure-sdk"],
+  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
+  "ignorePatterns": ["src/generated"],
```

Comment:
isn't this redundant with the entry in `.eslintignore`?

Reply:
Good point! Thank you!

Reply:
Done here! https://github.com/Azure/azure-sdk-for-js/pull/9668/commits/8347ac7c55470ef2fe46564ccbc0fd3234c73a7a